<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>hg FileViewer</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>33</y>
     <width>481</width>
     <height>405</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <property name="margin" >
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="HgRepoView" name="tableView_revisions" >
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid" >
        <bool>false</bool>
       </property>
       <property name="gridStyle" >
        <enum>Qt::NoPen</enum>
       </property>
      </widget>
      <widget class="HgFileView" native="1" name="textView" />
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar_edit" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>33</height>
    </rect>
   </property>
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionClose" />
   <addaction name="actionReload" />
  </widget>
  <action name="actionClose" >
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReload" >
   <property name="text" >
    <string>Reload</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HgRepoView</class>
   <extends>QTableView</extends>
   <header>hgrepoview.h</header>
  </customwidget>
  <customwidget>
   <class>HgFileView</class>
   <extends>QWidget</extends>
   <header>hgfileview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
